recipes/conda-forge-pinning/conda_build_config.yaml

```diff
@@ -0,0 +1,215 @@
+blas_impl:
+  - mkl                        # [x86 or x86_64]
+  - openblas
+arpack:
+  - 3.5
+boost:
+  - 1.65
+bzip2:
+  - 1.0
+cairo:
+  - 1.14
+# This differs from target_platform in that it determines what subdir the compiler
+#    will target, not what subdir the compiler package will be itself.
+#    For example, we need a win-64 vs2008_win-32 package, so that we compile win-32
+#    code on win-64 miniconda.
+cross_compiler_target_platform:  # [win]
+  - win-32                     # [win]
+  - win-64                     # [win]
+curl:
+  - 7.54
+c_compiler:                    # [win]
+  - vs2008                     # [win]
+  - vs2015                     # [win]
+  - vs2015                     # [win]
+cxx_compiler:                  # [win]
+  - vs2008                     # [win]
+  - vs2015                     # [win]
+  - vs2015                     # [win]
+fortran_compiler:              # [win]
+  - intel-fortran              # [win]
```

Comment:
Are we able to use this in `conda-forge`?

Reply:
It's an open package, yes, but this won't do you much good, because it's a dumb activation script, not an actual package with compilers.  We'll need to replace that with flang when flang is ready.